lib/api/CResultNormalizer.cc

```diff
@@ -71,14 +72,26 @@ bool CResultNormalizer::handleRecord(const TStrStrUMap& dataRowFields) {
     std::string partition;
     std::string partitionValue;
     std::string person;
+    std::string personValue;
     std::string function;
     std::string valueFieldName;
     double probability(0.0);
 
-    bool isValidRecord = parseDataFields(dataRowFields, level, partition, person,
+    // As of version 6.5 the 'personValue' field is required for (re)normalization to succeed.
+    // In the case of renormalization the 'personValue' field must be included in the set of 
+    // parameters sent from the Java side ML plugin to Elasticsearch.
+    // In production the version of the native code application and the java application it is communicating directly
+    // with will always match so supporting BWC with versions prior to 6.5 is not necessary.
+    bool isNormalizable = this->parseDataFields(dataRowFields, level, partition,
+                                         partitionValue, person, personValue,
```

Comment:
This doesn't look like it has been reformatted. Might be worth reformatting all files in this change again.